pubsub/test/iam.test.js

```diff
@@ -20,18 +20,19 @@ var policyMock = 'policy';
 var permissionsMock = 'permissions';
 
 function getSample () {
+  var apiResponseMock = {};
   var subscriptionMock = {
     iam: {
       getPolicy: sinon.stub().callsArgWith(0, null, policyMock),
```

Comment:
Actually, let me. I found like 6 methods so far (not just Pub/Sub ones) that don't return `apiResponse`. I'll make one big issue.